bool btn1_state
bool btn2_state
bool btn3_state
